migration/migration.go

```diff
@@ -229,6 +229,7 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB, witr *models.GormWork
 	if err := createOrUpdatePlannerItemExtention(models.SystemBug, ctx, witr, db); err != nil {
 		return err
 	}
+	witr.ClearCache() // Clear the WIT cache after updating existing WITs
```

Comment:
I agree with your sentiment here, but I think that ship has sailed: the models package boundary is not a chokepoint for controlling access to the set of known work item types. The migration package manipulates the set of work item types and therefore will need to control the items in the cache. Right now, the solution with the clear after doing all the updates will work, because we only ever read every work item type once. I think it's a better design to make this dependency explicit by either clearing the cache after every "loadTypeFromDB" (I'm not advocating this!) or by updating the cache when we update a type.